pytests/tests/test_signature.py

```diff
@@ -0,0 +1,90 @@
+#
+# Copyright (C) 2019 - 2020 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the GNU General Public License v2 (the "License");
+# you may not use this file except in compliance with the License. The terms
+# of the License are located in the COPYING file of this distribution.
+#
+#   Author: Siddharth Chandrasekaran <csiddharth@vmware.com>
+
+import os
+import re
+import tempfile
+import pytest
+
+DIST = os.environ.get('DIST')
+if DIST == 'photon':
+    DEFAULT_KEY='file:///etc/pki/rpm-gpg/VMWARE-RPM-GPG-KEY'
+elif DIST == 'fedora':
+    DEFAULT_KEY='file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-34-primary'
+
+@pytest.fixture(scope='function', autouse=True)
```

Comment:
Why is done at a function level? Is this intended?

Reply:
Yes, this is intentional. Cleanup is needed for each individual test.

Reply:
Okay.